### ISSUE TITLE
How to customize the javascript of the tenant ui module

### ISSUE THREAD

**Issue by imxiongstu** [abpframework/abp#15052]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Hi
I now have the following problems，I use the official mvc template，but I want to customize the js script of the tenant management page，I have set Index Js to place the “Pages\TenantManagement\Tenants\” directory,Display normally when running locally，but  after release/debug of publish,invalid，Check the corresponding js online，It was found that it was still official. The one I replaced didn't work，I have read many official documents and consulted with relevant communities, but failed.If possible, we need official support.
To sum up：Customization succeeded locally, but failed after publishing.
thanks.

### Reproduction Steps

1.Embedded
2.Added to virtual file

### Expected behavior

Can replace the official JS with your own.

### Version

6.0.0

### User Interface

MVC

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

**Comment by imxiongstu** [comment]
您的邮件我已经收到，请耐心等待回复熊www.imxiongstu.cn

**Comment by imxiongstu** [comment]
您的邮件我已经收到，请耐心等待回复熊www.imxiongstu.cn
